recipes/libgit2/all/conanfile.py

```diff
@@ -151,3 +151,5 @@ def package_info(self):
             self.cpp_info.system_libs.extend(["winhttp", "rpcrt4", "crypt32"])
         if self.settings.os == "Linux" and self.options.threadsafe:
             self.cpp_info.system_libs.append("pthread")
+        if self.settings.os == "Macos":
+            self.cpp_info.system_libs.append("curl")
```

Comment:
Hi! where did you find that libcurl is required for 0.27.9? If it is required, so we should use the libcurl conan package instead.